tests/Resources/keywords_hitta_verksamhet.robot

```diff
@@ -0,0 +1,27 @@
+*** Settings ***
+Documentation
+Library  SeleniumLibrary
+
+*** Keywords ***
+
+Test searchbar With Input
+    Sleep                                 1s
+    Input Text                            id:SearchQuery      göteborg
+    Click Element                         id:SearchButton
+    ${allPageListLinks} =                 Get Webelements         css:.organizationsearchresultblock li h3 a
+    ${allLinksCount} =                    Get Length              ${allPageListLinks}
+
+    FOR                                         ${INDEX}                IN RANGE                                1           ${allLinksCount}
+        ${linkUrl} =                            Get Element Attribute   ${allPageListLinks}[${INDEX}]          href
+
+        #Check if status code is 200
+        ${statusCodeResult}                     Check Status Code       ${linkUrl}
+        Should Be True                          ${statusCodeResult}     true
+    END
+
+
+Test searchbar Without Input
+    Sleep                                 1s
+    Click Element                         id:SearchButton
+    Sleep                                 1s
+    Wait Until Page Contains Element      css:.vgr-search-list
```

Comment:
Ska vi inte kolla så att det blir några träffar i listan med?

Reply:
Bra tanke!

Reply:
@niklasternstedt skulle du fixa den här grejen innan jag mergar in det?